### ISSUE TITLE
multiple user accounts with same email id.

### ISSUE THREAD

**Issue by sharad22gupta** [WWBN/AVideo#2809]
We can create multiple accounts with same email id and user name, I think there should be a check at the time of account creation to check for unique username and if email id already exists.

**Comment by DanielnetoDotCom** [comment]
Are you sure you can create accounts with the same username?

**Comment by sharad22gupta** [comment]
with the same email id, I can
![Annotation 2020-04-04 142228](https://user-images.githubusercontent.com/15939963/78423069-5f4d9480-7681-11ea-8dbc-97000dd25b35.png)

**Comment by akhilleusuggo** [comment]
@DanielnetoDotCom This is the 5th time people requesting this feature .
Can you make this happens ? A check email when creating an account . Only 1 email should be on the email table .
And is not about ''forceLoginToBeTheEmail''

**Comment by DanielnetoDotCom** [comment]
@akhilleusuggo Maybe I will add it in the future

**Comment by akhilleusuggo** [comment]
@DanielnetoDotCom Thank you

**Comment by JoshWho** [comment]
Yes I can confirm that users can use the same email with different usernames. I honestly thought that was by design,  I actually have a few test accounts all under the same email.

**Comment by DanielnetoDotCom** [comment]
Try this update and do not forget to check this option

![image](https://user-images.githubusercontent.com/18009293/78572078-60a3cc00-77fd-11ea-85b8-7451bb5f5aef.png)
